specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-05-01/managedClusters.json

```diff
@@ -2431,76 +2160,81 @@
+          "description": "Possible values are any decimal value greater than zero or -1 which indicates the willingness to pay any on-demand price. For more details on spot pricing, see [spot VMs pricing](https://docs.microsoft.com/azure/virtual-machines/spot-vms#pricing)"
         },
         "tags": {
           "type": "object",
           "additionalProperties": {
             "type": "string"
           },
-          "description": "Agent pool tags to be persisted on the agent pool virtual machine scale set."
+          "description": "The tags to be persisted on the agent pool virtual machine scale set."
         },
         "nodeLabels": {
           "type": "object",
           "additionalProperties": {
             "type": "string"
           },
-          "description": "Agent pool node labels to be persisted across all nodes in agent pool."
+          "description": "The node labels to be persisted across all nodes in agent pool."
         },
         "nodeTaints": {
           "type": "array",
           "items": {
             "type": "string"
           },
-          "description": "Taints added to new nodes during node pool create and scale. For example, key=value:NoSchedule."
+          "description": "The taints added to new nodes during node pool create and scale. For example, key=value:NoSchedule."
         },
         "proximityPlacementGroupID": {
-          "$ref": "#/definitions/ProximityPlacementGroupID",
+          "type": "string",
           "description": "The ID for Proximity Placement Group."
         },
         "kubeletConfig": {
           "$ref": "#/definitions/KubeletConfig",
-          "description": "KubeletConfig specifies the configuration of kubelet on agent nodes."
+          "description": "The Kubelet configuration on the agent pool nodes."
         },
         "linuxOSConfig": {
           "$ref": "#/definitions/LinuxOSConfig",
-          "description": "LinuxOSConfig specifies the OS configuration of linux agent nodes."
+          "description": "The OS configuration of Linux agent nodes."
         },
         "enableEncryptionAtHost": {
           "type": "boolean",
-          "description": "Whether to enable EncryptionAtHost"
+          "title": "Whether to enable host based OS and data drive encryption.",
+          "description": "This is only supported on certain VM sizes and in certain Azure regions. For more information, see: https://docs.microsoft.com/azure/aks/enable-host-encryption"
         },
         "enableUltraSSD": {
           "type": "boolean",
           "description": "Whether to enable UltraSSD"
         },
         "enableFIPS": {
           "type": "boolean",
-          "description": "Whether to use FIPS enabled OS"
+          "description": "Whether to use a FIPS enabled OS."
```

Comment:
I think we probably need a link here, and/or more details. Do we have a good article talking about FIPS on AKS?

Reply:
```suggestion
          "description": "Whether to use a FIPS-enabled OS."
```

Reply:
Applied your suggestion @porges

Reply:
Also found and added a link to https://docs.microsoft.com/en-us/azure/aks/use-multiple-node-pools#add-a-fips-enabled-node-pool-preview